app/src/main/java/com/teamagam/gimelgimel/app/view/MainActivity.java

```diff
@@ -90,6 +87,9 @@
     @Inject
     SyncDataConnectivityStatusInteractorFactory mDataAlertsFactory;
 
+    @Inject
+    UserPreferencesRepository userPreferencesRepository;
```

Comment:
Its not private only due to di.
You're right that we don't have a convention for it, but the other injected field in this class is prefixed with m, so we could at least keep it consistent within the class :)